src/evict/evict_lru.c

```diff
@@ -1678,6 +1679,31 @@ __evict_walk_file(WT_SESSION_IMPL *session,
 	if (F_ISSET(session->dhandle, WT_DHANDLE_DEAD) ||
 	    target_pages > remaining_slots)
 		target_pages = remaining_slots;
+
+	/*
+	 * These statistics generate a histogram of the number of pages targeted
+	 * for eviction each round. The range of values here are targeted at the
+	 * likely ranges for eviction based on some initial testing. The aim
+	 * is to cover these likely ranges in as few statistics as possible to
+	 * reduce the overall overhead.
+	 */
+	if (target_pages < 16) {
```

Comment:
I think it's interesting to track how often this value is 0. It's also valuable to see finer granularity for low numbers - it gives good insight into how sampling is working when there are a lot of trees in cache.